### ISSUE TITLE
Prepare for React 16

### ISSUE THREAD

**Issue by mikecx** [dennisduong/react-tipsy#5]
Current the code is throwing a deprecation error and it would be nice to have this good to go.

```
Warning: Tipsy: React.createClass is deprecated and will be removed in version 16. Use plain JavaScript classes instead. If you're not yet ready to migrate, create-react-class is available on npm as a drop-in replacement.
```

**Comment by mikecx** [comment]
Looks like pull request #4 addresses this.

**Comment by dennisduong** [comment]
I've merged pull request #4. Thanks for the report
